### DISCUSSION TITLE
RTMP Ingress not working

### DISCUSSION THREAD

**Discussion by yasmagic** [mynaparrot/plugNmeet-server#611] [General]
I tried streaming to the app using RTMP ingress credentials and it seems not working. How to make this work? Why does it say Ingress type as '0'? Thank you.

![image](https://github.com/user-attachments/assets/dc3dd91e-c9d4-4049-aab2-6f28a6d99ed4)

**Comment by jibon57** [comment]
fix: https://github.com/mynaparrot/plugNmeet-server/commit/2fc55a4e5c02787962a4e77804cfb83b38c7d975
Will you please try again?

**Comment by yasmagic** [comment]
I tested again by rebuilding the server. Now I can see in logs that the ingress stream is reaching server correctly but the video doesn't appear in the frontend app.

plugNmeet[6261]: livekit-ingress-1  | 2024-12-14T20:34:30.608Z#011INFO#011ingress#011stats/media.go:169#011track stats update#011{"nodeID": "NE_c239E6xwGb8D", "ingressID": "IN_VLJ5UB6ru5nu", "resourceID": "RT_gWY95ddARoDb", "roomName": "room15", "participantIdentity": "1734208375167", "ingressID": "IN_VLJ5UB6ru5nu", "resourceID": "RT_gWY95ddARoDb", "roomName": "room15", "participantIdentity": "1734208375167", "name": "output.video.HIGH", "currentBitrate": 1757056, "averageBitrate": 1757056, "currentPackets": 1755, "totalPacket": 1755, "currentLossRate": 0, "totalLossRate": 0, "currentPLI": 0, "totalPLI": 0, "jitter": "p50:0.973383 p90:1.9623226666666667 p99:2.1718086"}
